keras/backend/tensorflow_backend.py

```diff
@@ -3632,3 +3644,120 @@ def foldr(fn, elems, initializer=None, name=None):
         Same type and shape as initializer
     """
     return tf.foldr(fn, elems, initializer=initializer, name=name)
+
+
+def dropout_on_input(inputs, dim, timesteps, dropout_value, training):
```

Comment:
The original purpose of this function is because cntk have issue when repeat the dropout with time-step axis when timesteps is not a specified dimension, so we create this function to wrapper the difference. After discussion, we decide to use a workaround in cntk backend, so now we don't need this change.